__tests__/app.test.js

```diff
@@ -56,6 +56,25 @@ describe('ripe-banana routes', () => {
       });
   });
 
+  it('returns all actors using the GET route /actors', () => {
+    return request(app)
+      .get('/actors')
+      .then((response) => {
+        expect(response.body).toEqual(expect.arrayContaining([{ actorId: expect.any(String), name: expect.any(String)}]));
+        expect(response.body.length).toEqual(20);
```

Comment:
Nice use of an additional expect statement to help verify the validity of your response data!